docs/articles/documentation/using-testcafe/programming-interface/Global.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+title: Global Members
+permalink: /documentation/using-testcafe/programming-interface/global/
+---
+# Global Members
```

Comment:
"Global" is incorrect word for this. Globals is the global variable, available across all modules: https://nodejs.org/api/globals.html#globals_global_objects